README.md

```diff
@@ -41,8 +41,10 @@ var foods = [ ["apple","banana","strawberry"], ["pizza","fries","hamburger"] ];
 
 Create a For Loop that outputs the following string for each piece of fruit in the console. "I want to eat a [fruit]"
 
-```js
-// write code here
+```
+for (i=0; i<foods.length; i++) {
+    console.log("I want to eat a "+foods[i]);
+}
 ```
```

Comment:
This prints out the two arrays of fruit within `foods`. If we're going to print out fruits individually, we'll need to run another loop inside your existing one.